extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -123,8 +123,14 @@ function getState(element, mu) {
       'id': section.getAttribute('id'),
       'onExpandStateChange': expandStateShim,
     });
-    const headerProps = dict({'as': headerShim});
-    const contentProps = dict({'as': contentShim});
+    const headerProps = dict({
+      'as': headerShim,
+      'id': section.firstElementChild.getAttribute('id'),
```

Comment:
We need to add `id` as an `attributeFilter` when the mutation observer is initialized.

Reply:
Added in latest revision.